package-system/OpenSSL/set_openssl_port_to_1_1_1_m.patch

```diff
@@ -1,70 +1,63 @@
-From 66c310b442ac237b4b7e8b733eea83e94621809f Mon Sep 17 00:00:00 2001
-Date: Thu, 15 Apr 2021 09:02:46 -0700
-Subject: [PATCH] Set OpenSSL version to 1.1.1b
+From 18884b079e3945b29e4adc3b33a76c19929f9b0a Mon Sep 17 00:00:00 2001
```

Comment:
does the date after the hash ("Mon Sep 17...") need to be updated as well, or is this autogenerated?

Reply:
That is autogenerated from git.